entries/QUnit.module.xml

```diff
@@ -15,16 +15,28 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="contained_suite" optional="true">
```

Comment:
Is "suite" a QUnit concept? If not, all such references should probably be replaced with "module".